jpegint.h

```diff
@@ -47,6 +47,19 @@ typedef enum {            /* Operating modes for buffer controllers */
 /* JLONG must hold at least signed 32-bit values. */
 typedef long JLONG;
 
+/*
+ * JUINTPTR must be able hold pointer values. Once C99 is required this can
+ * be replaced with uintptr_t from ≤stdint.h>.
+ */
+#ifdef __UINTPTR_TYPE__
```

Comment:
I can also use a `__STDC_VERSION__` check + `<stdint.h>` include if you prefer?

Reply:
No, I think `__UINTPTR_TYPE__` is the best approach.